test/common_nn.py

```diff
@@ -47,14 +47,14 @@ def get_weight(m):
         module_name='Linear',
         constructor_args=(10, 8),
         input_size=(4, 10),
-        reference_fn=lambda i, p: torch.mm(i, p[0].t()) + p[1].view(1, -1).expand(4, 8),
+        reference_fn=lambda i, p, m: torch.mm(i, p[0].t()) + p[1].view(1, -1).expand(4, 8),
```

Comment:
the standard in python is to use "_" if you aren't going to use the parameter (throughout this change).

Reply:
Done